apps/prairielearn/elements/pl-order-blocks/dag_checker.py

```diff
@@ -156,15 +169,19 @@ def grade_dag(submission, depends_graph, group_belonging):
     return min(top_sort_correctness, grouping_correctness), graph.number_of_nodes()
 
 
-def is_vertex_cover(G, vertex_cover):
+def is_vertex_cover(G: nx.DiGraph, vertex_cover: Iterable):
```

Comment:
```suggestion
def is_vertex_cover(G: nx.DiGraph, vertex_cover: Iterable[Any]) -> bool:
```

Reply:
is there a reason to explicitly declare `[Any]` ?

Reply:
More explicit is usually better. I'm not sure if this should be strings only though? Would be nice to have that explicitly based on how this works.

Reply:
Most generally, it should be the same type of the nodes in the `nx.DiGraph`, but `nx.DiGraph` doesn't allow type parameters. I'll go ahead and mark it `str` though because it always is that for now anyway.

I just wish I could also mark all the `nx.DiGraph`s as `nx.DiGraph[str]`, but I can't.